src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -329,25 +344,34 @@ class TimeRangeSelector extends React.PureComponent {
                 isAbsoluteSelected={isAbsoluteSelected}
               >
                 <SelectorList isAbsoluteSelected={isAbsoluteSelected}>
-                  {shouldShowRelative && (
-                    <RelativeSelector
-                      onClick={this.handleSelectRelative}
-                      selected={relative || DEFAULT_STATS_PERIOD}
-                    />
-                  )}
-                  {shouldShowAbsolute && (
-                    <SelectorItem
-                      onClick={this.handleAbsoluteClick}
-                      value="absolute"
-                      label={t('Absolute Date')}
-                      selected={isAbsoluteSelected}
-                      last={true}
-                    />
-                  )}
+                  <SelectorListHook
+                    isAbsoluteSelected={isAbsoluteSelected}
+                    relativeSelected={relativeSelected}
+                    handleSelectRelative={this.handleSelectRelative}
+                    handleAbsoluteClick={this.handleAbsoluteClick}
+                    shouldShowRelative={shouldShowRelative}
+                    shouldShowAbsolute={shouldShowAbsolute}
+                  >
+                    {shouldShowRelative && (
```

Comment:
Can you actually pull the children here out up into the `defaultComponent` of the `SelectorListHook`. Then at least it's a bit more clear that all the props being passed into the `SelectorListHook` are used.